hc/accounts/tests/test_switch_team.py

```diff
@@ -11,21 +11,22 @@ def test_it_switches(self):
         self.client.login(username="bob@example.org", password="password")
 
         url = "/accounts/switch_team/%s/" % self.alice.username
-        r = self.client.get(url, follow=True)
-
-        ### Assert the contents of r
+        r= self.client.get(url, follow=True)
+        self.assertIn("<title>My Checks - healthchecks.io</title>\n", r.content)
+        ### **Assert the contents of r **
 
 
     def test_it_checks_team_membership(self):
         self.client.login(username="charlie@example.org", password="password")
 
         url = "/accounts/switch_team/%s/" % self.alice.username
```

Comment:
The PR diff clearly shows that you are not the author of this line of code however, I recommend that you update this line and others that may be using `%` for string formatting to use `str.format()`.

Resource: [pyformat.info](https://pyformat.info/)